src/app/modules/consultations/consultation-profile/profile-card/profile-card.component.scss

```diff
@@ -259,8 +256,8 @@
 
   }
 
-  .divider {
-    border: 1px solid #CAE5F5;
-    margin: 16px -24px;
+  .padding{
```

Comment:
Why is this class renamed?

Reply:
Bro it's been added, basically I had removed the padding given to the parent container, and added padding class to each sections.

Reply:
@Bandinikhil - This name is misleading. This class not only adds padding but also adds a border. The class name `divider` was more accurate. Can you help me understand why it is called `padding`?

Reply:
@gpalsingh - Bro I thought, this class gives the padding to the each section with the border-bottom, so I had named it as padding. Sure bro I'll rename it as divider only.